src/test/java/com/wikia/webdriver/testcases/portableinfoboxtests/PortableInfoboxTests.java

```diff
@@ -44,13 +44,12 @@
  */
 public class PortableInfoboxTests extends NewTestTemplate {
 
-  Credentials credentials = Configuration.getCredentials();
   private final static Logger LOGGER = Logger.getLogger(PortableInfoboxTests.class.getName());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)